scripts/launch_cluster.py

```diff
@@ -0,0 +1,640 @@
+                    },
+                    {
+                        "ParameterKey": "NodeInstanceType",
+                        "ParameterValue": instance_type
+                    },
+                    {
+                        "ParameterKey": "KeyName",
+                        "ParameterValue": keypair_name
+                    },
+                    {
+                        "ParameterKey": "VpcId",
+                        "ParameterValue": vpc_id
+                    },
+                    {
+                        "ParameterKey": "Subnets",
+                        "ParameterValue": ",".join(vpc_subnet_ids)
+                    }
+                ],
+                TimeoutInMinutes=15,
+                OnFailure='DELETE'
+            )
+
+            if response is None:
+                click.echo("Response is None, create worker group stack failed.")
+                sys.exit(1)
+            if not "StackId" in response:
+                click.echo("StackId not in response, create worker group stack failed.")
+                sys.exit(1)
+            click.echo("Initiated workers (ETA 5-20 mins)...")
+            click.echo("Waiting for workers stack creation to complete...")
+
+            try:
+                # This is a water which waits for the stack deployment to complete.
+                waiter = self._cf.get_waiter('stack_create_complete')
+                res = waiter.wait(
+                    StackName=workers_name
+                )
+            except Exception as e:
+                click.echo("Error: %s, gave up waiting for stack to create." % str(e))
+                sys.exit(1)
+            click.echo("Worker stack created.")
+
+        stack = self._cf.describe_stacks(StackName=workers_name)
+        node_instance_role=None
+        # We need NodeInstanceRole output.
+        for v in stack["Stacks"][0]["Outputs"]:
+            if v["OutputKey"] == "NodeInstanceRole": node_instance_role=v["OutputValue"]
+
+        click.echo("Node instance role: %s" % node_instance_role)
+        click.echo("*** Update worker auth.")
+
+        config = "apiVersion: v1\nkind: ConfigMap\nmetadata:\n  name: aws-auth\n" + \
+                "  namespace: kube-system\ndata:\n  mapRoles: |\n" + \
+                "    - rolearn: " + node_instance_role + "\n" + \
+                "      username: system:node:{{EC2PrivateDNSName}}\n" + \
+                "      groups:\n        - system:bootstrappers\n        - system:nodes\n"
+
+        click.echo("Write config map...")
+        worker_auth_file =  os.path.dirname(self._output_path) + "/aws-auth-cm.yaml"
+        with open(worker_auth_file, "w") as f:
+            f.write(config)
+        
+        # add the worker resource to cluster
+        try:
+            subprocess.run(["kubectl", "--kubeconfig=%s" % self._output_path, "apply",
+                           "-f", worker_auth_file], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        except Exception as e:
```

Comment:
The error info is in `e.stderr`. ref:
https://github.com/alibaba/GraphScope/blob/10f36b1dee462bf3efe9439a354e4f9e6d9f6da1/k8s/precompile.py#L46-L47

Reply:
fixed.